### PR TITLE
Fix typo in progress bar docs

### PR DESCRIPTION
**Pull request by awaelchli**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure to update the docs?   
- [ ] Did you write any new necessary tests?  
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## What does this PR do?
Fixes # (issue).

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: pytorch_lightning/callbacks/progress.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def __init__(self):
                 self.enabled = True
 
             def disable(self):
-                self.enableenabled = False
+                self.enabled = False
```
Comment (Borda) [inline pytorch_lightning/callbacks/progress.py:30]:
```suggestion
                self.enable = False
```

File: pytorch_lightning/callbacks/progress.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ def __init__(self):
                 self.enabled = True
 
             def disable(self):
-                self.enabled = False
+                self.enable = False
```
Comment (awaelchli) [inline pytorch_lightning/callbacks/progress.py:30]: @Borda but it also needs to be changed above. I will change it. please don't merge yet

Comment (Borda) [reply pytorch_lightning/callbacks/progress.py:30]: sure, sorry, just chack it with master code... and missed the example above

**Pull request merged**